Fix build w/ recent gcc-16's libstdc++

Same issue as https://bugs.gentoo.org/971287 which only happens with gcc-16 libstdc++ snapshots.

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
--- a/chromium/third_party/blink/renderer/platform/widget/input/prediction/predictor_factory.cc
+++ b/chromium/third_party/blink/renderer/platform/widget/input/prediction/predictor_factory.cc
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.

+#include <climits>
+
 #include "third_party/blink/renderer/platform/widget/input/prediction/predictor_factory.h"

 #include "third_party/blink/public/common/features.h"
